app/shoes_app.rb

```diff
@@ -55,7 +55,7 @@ def home
         destination = ShoesApp.navigation.pop(true)
         app.visit destination
       rescue ThreadError
-        # Empty queue, no worries...
+        Shoes.logger.info "Empty queue:\n#{e.message}"
```

Comment:
This is actually going to log a ton of messages as this queue is more often empty than not. Could we do something like https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#disabling-cops-within-source-code to ignore this specific case from code?